server/models/handlers/account.js

```diff
@@ -1,9 +1,24 @@
 /* The goal: Take in the user input from the Account Creation form in 
 the front end UI in order to create a user account in the backend*/
 
-async function createUserAccount(firstName, lastName, type, studentID, email, password, phone) {
```

Comment:
Hi Nancy, 
Same comment as mentioned above, I would recommend adjusting the method parameter naming to lowerCamelCase.

Reply:
Done :)